### ISSUE TITLE
Determine whether attorneys can find advanced search and utilize its capabilities

### ISSUE THREAD

**Issue by JonellaCulmer** [18F/fec-testing#99]
At the DC FEC Conference, our goal for testing the AO/MUR boolean search was to determine if users could identify the options to conduct that search using the "More keyword options" criteria. 

We unfortunately, did not get enough data to move forward. Therefore, we will continue to test at the COGEL 2017 conference. Our goal at COGEL will be to determine whether attorneys both find these advanced options, and whether it is effectively usable. Testing with attorneys at COGEL, many of which may be familiar with Westlaw, will give us an opportunity to test with those who frequently use boolean type search to filter their search results. 

Completion criteria:
- [x] Refine testing goals and hypothesis
- [x] Write test script questions

**Comment by JonellaCulmer** [comment]
AO/MUR boolean search was part of the research done at COGEL. See test script below:

# December 3-6, 2017 Test Script
 Quick connection links:
Notetaking Link:
https://docs.google.com/document/d/1m6mk4ITQ-PVKCUilPDqT7puXajy0Ti3dicrvJSMaIcY/edit#heading=h.tn8vr3fhlrey 
Mural board:
https://app.mural.ly/t/gsa6/m/gsa6/1512484140409/4e647829e2d266b85767e01ad8e430a04846781c 
## Features to test:
### Legal resources landing page 
Tested on stage: https://fec-dev-proxy.app.cloud.gov/  
Landing page clarity
Ability to find compliance information
Ability to find recent legal news
Usability of global legal resources search
### AO/MUR boolean search 
Ability to find advanced boolean search options
Usability of ‘More keyword options’ 
### Audit search 
Tested via prototype: 
Navigating to the audit search
Conduct an audit search
### Open-ended feedback 
User-generated feedback on navigation, usability issues


## Introduction script:
*Start by asking how much time they have. That will help gauge how many mini-tests you can conduct. It will also reflect well on us, having taken the time to ask and respecting their time.

Hi, I'm ___, and I'm working on the Federal Election Commission website redesign, and we'd like to get some quick feedback on new features and content we're adding. Before we start, I want to make it clear that we're testing the designs, not you. You can't do anything wrong here and all your answers will be the right ones. Also, please don't worry that you're going to hurt our feelings. We're doing this to improve the site, so we need to hear your honest reactions.
Although it may seem a little weird, I'm going to ask you to think out loud as much as possible as you look at the designs. Describe what you're looking at, what you're trying to do, and what you're thinking. This will be a huge help to us. Sound good?
[wait for user confirmation; clarify things if user asks questions]
(credit: Krug)
## Introduction questions:
What is your job in campaign finance? [specify whether at federal, state or local level]
How long have you been in the industry?
## Topic questions:
### Legal Resources landing page
This test needs to be done on dev. Have user go to:
https://fec-dev-proxy.app.cloud.gov/  



Task 1: Landing page clarity
Looking at this design but WITHOUT clicking anything, what is your first impression? Do you see anything you would use?
Task 2: Ability to find compliance information
Pretend you’re advising a corporate PAC client. They’ve asked you about solicitations outside their restricted class, and you want to send them some compliance resources. Without clicking, where would you go to find that information?

Task 3: Ability to find recent legal news
The commission has just issued a new advisory opinion and you want to learn more about it. Do you see anything here you can use to learn more about the AO?

Task 4: Usability of global legal resources search
Imagine you’re researching the rules regarding foreign nationals. Do you see anything here that you could use to accomplish this goal?
Do you have any other feedback on this page you’d like to share?

### AO/MUR boolean search 
This test needs to be done on dev or production Here:
https://fec-dev-proxy.app.cloud.gov/ or here: www.fec.gov 


Task 1: Ability to find advanced boolean search options
Imagine you’ve been researching rules on contributions and would now like to refine your search to look for advisory opinions on foreign nationals. How would you conduct this search?
Task 2: Usability of ‘More keyword options’ 
Imagine you now want to refine your search further by looking for AOs on foreign nationals, but excluding AOs on U.S. citizens living abroad? How would you conduct that search?
### Audit search
This test needs to be done via prototype. Here: https://gsa.invisionapp.com/share/YBDVX4554#/screens/266479723 

So now let’s look at a prototype that hasn’t been built on the site yet. 

First, do you use audits as part of your work?

Task 1: Navigating to the audit search
Imagine you want to conduct an audit search now. What is the path you would take to conduct a search?
Task 2: Conduct an audit search [starting from the advanced audit search page]
Looking at this new design, how would you begin your search?
Imagine you wanted to search for committees with audits mentioning excessive contributions. Without clicking, how would you search for those audits?
Go to Gary Johnson - 2012. Can you find the final audit report?
What are your general impressions of this audit report page?
Is there anything you expected to see, but did not?

### Open-ended feedback:
Task 1: User-generated feedback
How often do you visit FEC.gov?
When you come to FEC.gov, what kinds of information are you looking for?
Can you try to do that now?
## Closing questions:
Is there anything you wished I asked you about, but didn’t?
Any last feedback you’d like to share?
Do you have any questions for me?
Before we go, I want to highlight the feedback tool we have on the website. We actively review all the feedback that’s received through the feedback tool and want to encourage you to use it as well if you see something that’s not right, or doesn’t work for you.

**Comment by JonellaCulmer** [comment]
We tested the AO/MUR search at the COGEL conference and the results were positive. The vast majority of those tested had no issue in finding the 'More keyword options' filter to conduct a more advanced search using boolean logic. 

There were, however, differences in the way that users conducted those searches. A minority of those tested were unsure when to use which search filter (ex. All of these words, Any of these words, This exact phrase, or None of these words). Because such a small group of users had this issue, this is now just something to keep an eye on moving forward in the event more users find they have trouble with the boolean filters. 

Additionally, others we tested with used symbols in the boolean filters boxes when it was necessary. We need to explore new language on the use of the filters, modify the way information is parsed - including stripping symbols (such as AND), or a combination of the two.
Closing in favor of new issue: (TBD)
